### PR TITLE
Add test about enip stats with _udp prefix

### PR DESCRIPTION
**Pull request by catenacyber**
## Ticket

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/6304

Suricata PR coming

### CODE REVIEW

File: tests/enip-stats-udp/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Description
+
+Test ENIP stats always have `_udp` or `_tcp` prefix
```
Comment (victorjulien) [inline tests/enip-stats-udp/README.md:3]: I don't see how this is tested?

Comment (catenacyber) [reply tests/enip-stats-udp/README.md:3]: As CI failure with current master shows

Comment (victorjulien) [reply tests/enip-stats-udp/README.md:3]: I expect a check for the stats by name, as that is what the work is about I think

Comment (catenacyber) [reply tests/enip-stats-udp/README.md:3]: Right, doing a next iteration.

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata-verify/pull/1546

**Pull request closed without merging**